presidio-analyzer/presidio_analyzer/nlp_engine/__init__.py

```diff
@@ -1,12 +1,64 @@
 """NLP engine package. Performs text pre-processing."""
+from typing import Union
 
-from .nlp_artifacts import NlpArtifacts  # noqa: F401
-from .nlp_engine import NlpEngine  # noqa: F401
-from .spacy_nlp_engine import SpacyNlpEngine  # noqa: F401
-from .stanza_nlp_engine import StanzaNlpEngine  # noqa: F401
+from .nlp_artifacts import NlpArtifacts
+from .nlp_engine import NlpEngine
+from .spacy_nlp_engine import SpacyNlpEngine
+from .stanza_nlp_engine import StanzaNlpEngine
+from .. import PresidioLogger
 
 _all_engines = [SpacyNlpEngine, StanzaNlpEngine]
 
+
 NLP_ENGINES = {
     engine.engine_name: engine for engine in _all_engines if engine.is_available
 }
+
+logger = PresidioLogger("presidio-analyzer")
+
+
+def create_nlp_engine(
+    nlp_configuration: dict,
+) -> Union[SpacyNlpEngine, StanzaNlpEngine]:
+    """
+    Create an NLP engine instance given a configuration file.
+
+    :param nlp_configuration: Dict containing configuration.
+    Example configuration:
+    {
+        "nlp_engine_name": "spacy",
+        "models": [{"lang_code": "en",
+                    "model_name": "en_core_web_lg"
+                  }]
+    }
+    """
+
+    if (
+        not nlp_configuration
+        or not nlp_configuration["nlp_engine_name"]
```

Comment:
would't this raise a KeyError? either nlp_configuration.get("nlp_engine_name") or "nlp_engine_name" not in nlp_configuration

Reply:
Add tests on this scenario :)

Reply:
Fixed + added tests